src/tools/CMakeLists.txt

```diff
@@ -52,11 +52,9 @@ if(WITH_TESTS)
 endif(WITH_TESTS)
 endif(WITH_LIBCEPHFS)
 
-if(WITH_TESTS)
```

Comment:
kvstore-tool is packaged to `ceph-test`(user need to install `ceph-test` by `apt install ceph-test`)....here is a little hack, but i don't have a better solution. script testing could not find `ceph-kvstore-tool` if we do not enable `ceph-kvstore-tool` building by default.

Reply:
seems fine to me